### PR TITLE
color decoder added

### PR DESCRIPTION
**Pull request by Tjerk258**
added color decoder from script to numbers

### CODE REVIEW

File: Core/Src/API.c [hunk 1]
```diff
@@ -202,14 +203,14 @@ void drawLines(uint16_t x_begin, uint16_t y_begin, uint16_t x_eind, uint16_t y_e
 
 void drawRect(uint16_t x_pos, uint16_t y_pos, uint16_t length, uint16_t width, uint8_t kleur, uint8_t filled)
 {
-	uint8_t i = 0;
+	uint16_t i = 0;
```
Comment (bestehustudentEU) [inline Core/Src/API.c:206]: was did change needed?

**Review by Shariefie — APPROVED** [review]
Code reviewed.

File: Core/Src/logic_layer.c [hunk 2]
```diff
@@ -49,40 +124,40 @@ void logic_layer(char commando[])
...
 						(uint16_t)atoi((char*)commando_filled[4]),
-						(uint16_t)atoi((char*)commando_filled[5]),
+						(uint16_t)kleur_decoder((char*)commando_filled[5]),
 						(uint16_t)atoi((char*)commando_filled[6]));
 				break;
-			/*case 2:
-				drawLines(100,10,200,10,0x1F,5);
-				/*drawText((uint16_t)atoi((char*)commando_filled[1]),
+			case 2:
+//				drawLines(100,10,200,10,0x1F,5);
```
Comment (bestehustudentEU) [inline Core/Src/logic_layer.c]: delete this line!

**Review by bestehustudentEU — COMMENTED** [review]
good enough for git, not good enough to present it.

File: Core/Src/API.c (hunk 1 above)
Comment (Tjerk258) [reply Core/Src/API.c:206]: yes because for filling te rectangle

File: Core/Src/logic_layer.c (hunk 2 above)
Comment (Tjerk258) [reply Core/Src/logic_layer.c]: deleted

Comment (Tjerk258) [reply Core/Src/logic_layer.c]: LiarLiarLiarMovieGIF

**Pull request merged**